docs/src/main/asciidoc/legal.adoc

```diff
@@ -1 +1,4 @@
+
+(C) 2016-2021 the original authors.
```

Comment:
2016-2021 --> 2016-2022?

Reply:
what's this ?

Reply:
https://docs.spring.io/spring-framework/docs/current/reference/html/

![image](https://user-images.githubusercontent.com/13167207/149290506-14c98314-9a5b-4106-b477-53614821943d.png)